community/2023-02-01-community-meeting.md

```diff
@@ -0,0 +1,21 @@
+---
+title: 2023-02-01 Community Meeting
+authors: [brooks]
```

Comment:
@ricochet if you want to add yourself into the authors.yml under community, feel free to change this to you!